modular_skyrat/modules/conveyor_sorter/code/conveyor_sorter.dm

```diff
@@ -1,12 +1,13 @@
 /obj/item/conveyor_sorter
 	name = "conveyor sorter lister"
-	desc = "A tool that is used to not only create the conveyor sorters, but give lists to the conveyor sorters."
+	desc = "A tool that is used to not only create the conveyor sorters, but give lists to the conveyor sorters. It has a limit of four sorters"
```

Comment:
This isn't necessary, and is bad practice because you already added something for that to be displayed further down in the examine.
```suggestion
	desc = "A tool that is used to not only create the conveyor sorters, but give lists to the conveyor sorters."
```